client/dashboard/data/site.ts

```diff
@@ -31,6 +31,7 @@ export const SITE_OPTIONS = [
 	'admin_url',
 	'is_redirect',
 	'is_wpforteams_site',
+	'updated_at',
```

Comment:
Let's sort this alphabetically 🙈

Reply:
Oops, yes we should. cb2b40c02f97a85977be3a31ce9fd2fa1ac6461a